distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/ServiceManagerImpl.java

```diff
@@ -135,8 +135,6 @@ public void startManagedService(String servicePid, Map<String, Object> propertie
     @Override
     public void stopManagedService(String servicePid) throws IOException {
         Configuration sourceConfig = adminConfig.getConfiguration(servicePid, null);
-        ServiceManagerImpl.ServiceConfigurationListener listener =
```

Comment:
Tracy is right. It looks like the listener isn't working the way I thought it did and isn't being used right now in this method, so we can safely remove the line for now.

Having said that, I still think we have two potential issues here:

1. The `startManagedService` _should_ remove the listener once it's done using it. In other words, we should keep a reference to the `ServiceRegistration`here: https://github.com/codice/ddf/blob/master/distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/ServiceManagerImpl.java#L159, and call `ServiceRegistration.unregister` after the `while` loop here: https://github.com/codice/ddf/blob/master/distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/ServiceManagerImpl.java#L167-L176.
1. The `stopManagedService` should wait until the service is actually stopped before returning like `startManagedService` does.

We probably want to address the first point as part of this ticket. The second on is probably outside the scope of this ticket though, so we could open a separate ticket and address later. Unless you feel like tackling it right now.